README.md

```diff
@@ -59,12 +59,32 @@ to the Topic Taxonomy. It also supports tagging individual pages.
 
 ## Screenshots
 
+### Tagging content
```

Comment:
This heading isn't rendering correctly. It's being displayed as:

```
### Tagging content
```

rather than a heading.

Reply:
Hate when that happens! Good spot.